docusaurus/docs/reference/tunnelers/linux/linux-tunnel-options.md

```diff
@@ -45,7 +45,7 @@ There are two run modes:
 
 ### `run` Mode
 
-`ziti-edge-tunnel run` provides a transparent proxy and nameserver. The nameserver may be configured to be authoritative (the default) or recursive with a command-line option. The OS is automatically configured to treat the nameserver as primary. You may inspect the resulting configuration with these commands.
+`ziti-edge-tunnel run` provides a transparent proxy and nameserver. The nameserver may be configured to be authoritative (the default) or recursive with command-line option `--dns-upstream`. `systemd-resolved`, if enabled, is automatically configured to treat the Ziti nameserver as primary. You may inspect the configuration with these commands.
```

Comment:
when running under systemd-resolved, we associate our name server with the tun interface. I wouldn't call this "primary" though. That sounds more like we are the system's global name server, similar to resolv.conf